docs/ml-migration-guide.md

```diff
@@ -20,15 +19,80 @@ license: |
   limitations under the License.
 ---
 
-The migration guide for the current Spark version is kept on the [MLlib Guide main page](ml-guide.html#migration-guide).
+* Table of contents
+{:toc}
 
-## From 2.1 to 2.2
+Note that this migration guide describes the items specific to MLlib.
+Many items of SQL migration can be applied when migrating MLlib to higher versions for DataFrame-based APIs.
+Please refer [Migration Guide: SQL, Datasets and DataFrame](sql-migration-guide.html).
+
+## Upgrading from MLlib 2.4 to 3.0
+
+### Breaking changes
+{:.no_toc}
```

Comment:
`{:.no_toc}` is needed to exclude in the "Table of Contents". For example:

Without no_toc
![Screen Shot 2019-09-11 at 7 30 08 PM](https://user-images.githubusercontent.com/6477701/64690137-a8ae7a80-d4ca-11e9-8ed6-109c08a30e8a.png)

With no_toc
![Screen Shot 2019-09-11 at 7 30 24 PM](https://user-images.githubusercontent.com/6477701/64690140-aa783e00-d4ca-11e9-843b-8d3bdec790d1.png)